scIB/integration.py

```diff
@@ -192,7 +192,88 @@ def runScvi(adata, batch, hvg=None):
     adata.obsm['X_emb'] = latent
 
     return adata
+def runScanvi(adata, batch, labels, hvg=None):
```

Comment:
`hvg` is not used in the function. Could remove this argument.

Reply:
that's true for almost every function I think

Reply:
legacy argument i guess... let's remove here though... as no reason to add to the legacy code.